packages/protocol/test/common/integration.ts

```diff
@@ -166,7 +173,13 @@ contract('Integration: Exchange', (accounts: string[]) => {
       await exchange.exchange(sellAmount, minBuyAmount, true)
       const finalGold = await goldToken.balanceOf(accounts[0])
 
-      assert.isTrue(finalGold.lt(originalGold))
+      const block = await web3.eth.getBlock('latest')
+      if (isSameAddress(block.miner, accounts[0])) {
```

Comment:
This issue again 🙄

Nice find, this can be a tricky one to debug. I wonder if there's a way to turn ganache block rewards off